src/c++/vdo/base/data-vio.c

```diff
@@ -808,8 +804,7 @@ static void process_release_callback(struct vdo_completion *completion)
 		wake_up_nr(&pool->limiter.blocked_threads, to_wake);
 
 	if (discards_to_wake > 0) {
-		wake_up_nr(&pool->discard_limiter.blocked_threads,
-			   discards_to_wake);
+		wake_up_nr(&pool->discard_limiter.blocked_threads, discards_to_wake);
 	}
```

Comment:
Putting a flag here so I remember to look at it again. This is the place where braces need to go away, now that the block is just a single line.